packages/types/src/connect/config.ts

```diff
@@ -3,6 +3,7 @@ import type { ConnectAppendError, ConnectLoginError } from './errors';
 export type CorbadoConnectLoginConfig = {
   onFallback(email: string): void;
   onError?(error: ConnectLoginError): void;
+  onHelpRequest?(): void;
```

Comment:
To test that this handler is also working, can we make use of it in the playground (a simple alert("help") is sufficient I think